docs/using/tutorial/vagrant-setup.rst

```diff
@@ -113,9 +121,21 @@ Note that you will need to make the same substitution in commands used throughou
    .. code-block:: console
 
       alice@mercury:~/flocker-tutorial$ ls
-      Vagrantfile
+      cluster.crt user.crt user.key Vagrantfile
```

Comment:
We haven't downloaded the certificates yet so this shell example is wrong.
I'd probably just remove it.